README.md

```diff
@@ -50,13 +50,24 @@ pipx run --spec phylum phylum-ci <options>
 These installation methods require Python 3.9+ to run.
 For a self contained environment, consider using the Docker image as described below.
 
-A Windows binary, `phylum-ci.exe`, is offered as a [release artifact][latest_rels] and does not require Python to run.
-Simply [download the latest version][latest_artifact] and run it to access the same
-[`phylum-ci` script entry point features][anchor_script].
+Windows binaries are offered as [release artifacts][latest_rels] for a "standalone" solution that does not require
+Python or Docker to run. There are two options for this installation method:
+
+* `phylum-ci.zip`
+  * [Download the latest version][latest_zip] and extract the archive
+  * Add the extracted directory to `PATH` or reference the contained `phylum-ci.exe` binary directly
+* `phylum-ci.exe`
+  * [Download the latest version][latest_exe] and place this binary on `PATH` or reference it directly
+  * This is a self-extracting executable that adds a version-specific directory in the local user cache
+  * An advantage to this solution is that it is a single file
+  * A disadvantage is that the file may trigger AV since it uses a packer and is not digitally signed
+
+Either Windows "installation" method allows access to the same [`phylum-ci` script entry point features][anchor_script].
```

Comment:
Are we quoting "installation" because the portable version isn't supposed to be considered "installed"?

Reply:
Yes. Both of these methods are really more "download and run" or "download, extract, and run" which are different than traditional Windows installers where the registry is updated and bunch of files are added to the system, with shortcuts/links, etc.